### PR TITLE
Add napari-cilia-beating-frequency recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@jaimergp, @emilmelnikov do you agree with being added as maintainers?

**Comment by goanpeca** [comment]
Have not heard from the author in a bit. Moving to ready for review.

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Comment by emilmelnikov** [comment]
Hi @goanpeca. Sorry for the late reply.

I'm a bit unsure about adding this package to conda-forge: it's probably of little value to people in this state. I've pushed it to PyPI so that my collaborator could easily install it via napari hub. Now I'm not sure this was the right way.

What is the recommended approach in such situations? Do you encourage people to push all packages to PyPI/conda-forge anyway, or is it better to send such plugins privately and install them manually?

**Comment by goanpeca** [comment]
Hi @emilmelnikov, 

>I'm a bit unsure about adding this package to conda-forge: it's probably of little value to people in this state. I've pushed it to PyPI so that my collaborator could easily install it via napari hub. Now I'm not sure this was the right way.

From what you mention, It seems your plugin is not ready to be distributed to the bigger napari community via hub. It is ok to publish to PyPi only and indicate your colleague to install it by typing the pypi name directly in the box as stated here https://napari.org/stable/plugins/find_and_install_plugin.html#installing-plugins-with-napari

However as you added the `Framework :: napari` (see https://pypi.org/search/?q=&o=&c=Framework+%3A%3A+napari) then napari-hub indexed it automatically.

We are migrating all plugins based on the information that napari-hub aggregates.

If you consider that your plugin is not ready to be shared with the wider napari-hub community, then I would advice you remove the `Framework :: napari` from the framework classifier and release a new version, that way it will no longer be displayed in the napari-hub. 

We will be adding more documentation :) 

Cheers!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-cilia-beating-frequency/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - napari_cilia_beating_frequency
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/napari-cilia-beating-frequency/
```
Comment (dopplershift) [inline recipes/napari-cilia-beating-frequency/meta.yaml:39]: This is 404-ing, so maybe the plugin was deleted?

Comment (dopplershift) [reply recipes/napari-cilia-beating-frequency/meta.yaml:39]: Ooops, I meant the "homepage" link on there: https://github.com/emilmelnikov/napari-cilia-beating-frequency

Comment (goanpeca) [reply recipes/napari-cilia-beating-frequency/meta.yaml:39]: Pinging @emilmelnikov :)

Comment (emilmelnikov) [reply recipes/napari-cilia-beating-frequency/meta.yaml:39]: Hi! Sorry for stalling this issue, I didn't have access to a laptop recently :)

As discussed earlier in https://github.com/conda-forge/staged-recipes/pull/19674#issuecomment-1220107061, I removed the `Framework :: napari` classifier, but forgot to push and upload the new version, sorry for that.

This plugin is actually a very early prototype, and the work on it stopped a while ago, so I don't think the wider community will benefit from it, at least in it's current state.

Comment (goanpeca) [reply recipes/napari-cilia-beating-frequency/meta.yaml:39]: Thanks for the heads up!

**Pull request closed without merging**